infra/scripts/cometbft.toml

```diff
@@ -21,7 +21,7 @@ docker run \
   --name ${CMT_CONTAINER_NAME} \
   --user $(id -u) \
   --network ${NETWORK_NAME} \
-  --publish 26657:${CMT_HOST_PORT} \
+  --publish ${CMT_HOST_PORT}:26657 \
```

Comment:
Oops, this is embarrassing :disappointed_relieved: 
Could I ask you to fix my mistakes [here](https://github.com/consensus-shipyard/fendermint/blob/2dc335ac1fc2c901f005017c8104c447bddb0f90/fendermint/testing/smoke-test/Makefile.toml#L132) and [here](https://github.com/consensus-shipyard/fendermint/blob/2dc335ac1fc2c901f005017c8104c447bddb0f90/fendermint/testing/smoke-test/Makefile.toml#L182) as well? :pray:

Reply:
I actually didn't want to touch it in case I was making a mistake myself and I was waiting for your review to confirm haha I have to admit that this bug was driving me crazy at first, but fixed :)